src/SerialLoops/Dialogs/BgmLoopPropertiesDialog.cs

```diff
@@ -0,0 +1,233 @@
+﻿using Eto.Forms;
+using HaruhiChokuretsuLib.Util;
+using NAudio.Wave;
+using SerialLoops.Controls;
+using SerialLoops.Lib.Items;
+using SerialLoops.Lib.Util.WaveformRenderer;
+using SerialLoops.Utility;
+using SkiaSharp;
+using System;
+using System.IO;
+
+namespace SerialLoops.Dialogs
+{
+    public class BgmLoopPropertiesDialog : Dialog
+    {
+        private ILogger _log;
+
+        public bool SaveChanges { get; set; } = false;
+        public BgmLoopPreviewItem LoopPreview { get; set; }
+        public SKGuiImage Waveform { get; set; }
+        public SoundPlayerPanel LoopPreviewPlayer { get; set; }
+
+        public BgmLoopPropertiesDialog(WaveStream wav, string title, ILogger log, bool loopEnabled, uint startSample = 0, uint endSample = 0)
+        {
+            _log = log;
+            Title = title;
```

Comment:
```suggestion
            Title = $"{title} - Manage Loop";
```
Just for consistency with other modals would be nice to have the titles be fully qualified